src/components/Box/Box.test.tsx

```diff
@@ -0,0 +1,52 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import Box, { BoxProps } from './index';
+
+describe('<Box />', () => {
+  let component: ShallowWrapper<BoxProps, {}>;
+
+  describe('with default options', () => {
+    beforeEach(() => {
+      component = shallow(<Box />);
+    });
+
+    it('has its correct base class', () => {
```

Comment:
i think we just changed to rely on snapshots instead of these individual tests, so you can remove all the individual `it` blocks except for the snapshot tests